introduction/site_survey.md

```diff
@@ -0,0 +1,404 @@
+   > would be the last hop route to the CAN network.
+
+<a name="default_internal_network_ranges"></a>
+
+## 5. Default internal network ranges
+
+The initial installation of the system creates default networks with default settings and with no external exposure.
+These default IP address ranges ensure that no nodes in the system attempt to use the same IP address as a
+Kubernetes service or pod, which would result in undefined behavior that is extremely difficult to reproduce or
+debug.
+
+The following table shows the default IP address ranges.
+
+| Network | Default IP Address Range | Site Value (if not default) |
+| --- | --- | --- |
+| Kubernetes service network | `10.16.0.0/12` | |
+| Kubernetes pod network | `10.32.0.0/12` | |
+| Install Network (MTL) | `10.1.0.0/16` | |
+| Node Management Network (NMN) | `10.252.0.0/17` | |
+| High Speed Network (HSN) | `10.253.0.0/16` | |
+| Hardware Management Network (HMN) | `10.254.0.0/17` | |
+| Mountain NMN allocate a `/22` from this range per liquid cooled cabinet | `10.100.0.0/17` | |
+| Mountain HMN allocate a `/22` from this range per liquid cooled cabinet | `10.104.0.0/17` | |
+| River NMN | `10.106.0.0/17` | |
+| River HMN | `10.107.0.0/17` | |
+
+> Note: Example NMN `10.100.0.0/17`and HMN `10.104.0.0/17` default IP address ranges for a mountain system with 3 cabinets would be:
+>
+> | Cabinet number | NMN Default IP Address Range | Site Value (if not default) |
+> | --- | --- | --- |
+> | 1 | 10.100.0.0/22 | |
+> | 2 | 10.100.4.0/22 | |
+> | 3 | 10.100.8.0/22 | |
+>
+> | Cabinet number | HMN Default IP Address Range | Site Value (if not default) |
+> | --- | --- | --- |
+> | 1 | 10.104.0.0/22 | |
+> | 2 | 10.104.4.0/22 | |
+> | 3 | 10.104.8.0/22 | |
+
+<a name="SHCD"></a>
+
+## 6. SHCD
+
+The Shasta Cabling Diagram (SHCD) is a multiple tab spreadsheet prepared by HPE Cray Manufacturing with much information about the
+HPE Cray EX system and its components. Included in the SHCD are:
+
+* A configuration summary with revision history
+* Floor layout plan with cabinet ID numbers
+* Type and location of components in the air-cooled cabinets
+* Type and location of components in the liquid-cooled cabinets
+* Device diagrams for switches and nodes in the cabinets
+* List of source and destination of every HSN cable
+* List of source and destination of every cable connected to the spine switches
+* List of source and destination of every cable connected to the NMN
+* List of source and destination of every cable connected to the HMN
+* List of cabling for the KVM
+* Routing of power to the PDUs
+
+The installation of CSM software requires that the SHCD be available. Some information will be manually collected from
+the SHCD, but some of the tabs can be extracted into CSV formatted files to be fed to automatic configuration tools.
+
+<a name="csi_command_line_configuration_payload"></a>
+
+## 7. `csi` command line configuration payload
```

Comment:
Do we really want to duplicate all of the information from the prepare_configuration_payload page on this page? That seems like a recipe for updating one place and forgetting to update the other. Can we not just have one of the two pages link to the other one?

Reply:
Need to have a single worksheet format for the site survey without links to other places.  This could be printed out and written upon.

I don't like having it in two places, but don't see a way to meet the need of the site survey.

Reply:
In that case, could we not modify the prepare_configuration_payload page to have it link here? Already the site survey page (which looks awesome, BTW) seems like the most comprehensive "one stop shop" overview of this stuff, so I think it would be fine to have this be the new home for most of this, and have the other things link over here. Or is that a bad idea?